### DISCUSSION TITLE
Wireguard LXC, autostart wg inteface after PVE reboot

### DISCUSSION THREAD

**Discussion by ekz0rcyst** [tteck/Proxmox#3632] [Help Needed]
Hi all!
After restart Proxmox node, and start Wireguard LXC (installed with helper script), need go to WG dashboard and turn on wg interface. How to make autostart?

**Comment by tteck** [comment]
It has a systemd service to start on boot

https://github.com/tteck/Proxmox/blob/6cb0bc59d9f177a99c77ccdd395403240cd077ea/install/wireguard-install.sh#L39-L61

**Comment by ekz0rcyst** [comment]
![Знімок екрана 2024-08-31 о 20 35 17](https://github.com/user-attachments/assets/7ed2dbe2-5e52-4184-b121-ad890ae05e2a)
What about this?

**Comment by ekz0rcyst** [comment]
After reboot Proxmox and starts VMs and LXCs, this checkbox is status off. And after put this checkbox and enable interface all working great. How to automated this? Sorry for noob question.

**Reply by Chrischi-** [reply]
I have encountered the same problem. After a restart, the checkbox is deactivated and therefore no connection is possible.

**Reply by Fridasbabe** [reply]
Jup, really annoying! This i critical for me to work, otherwise i going back to pivpn...

**Comment by shuperluu** [comment]
I solved on mine by login at LXC console and use - sudo systemctl enable wg-quick@CONFIGNAME (in my case wg1 as CONFIGNAME).

After a weekly backup the VPN was down and I found out at WGDashboard an issue is open for next development.

**Comment by stickpin** [comment]
Here is the official solution from the WGDashboard docs: [Enable WireGuard configuration when WGDashboard start﻿](https://donaldzou.github.io/WGDashboard-Documentation/usage.html#enable-wireguard-configuration-when-wgdashboard-start)
